packaging/flatpak/README.md

```diff
@@ -0,0 +1,40 @@
+# Flatpak packaging
+
+## Manifest
+
+Flatpak packaging manifest is generated by running
+
+```
+packaging/flatpak/generate-flatpak-script.sh
+```
+
+from the root of the project. This script requires `libolm` to be
+installed on the development PC as it will create Python virtual
+environment and install all the requirements in it.
+
+Note that the requirements are taken from
+`packaging/flatpak/requirements.flatpak.txt` and the ones specified in
+the project' root `requirements.txt`. At the moment of writing,
+included packages are `multidict` with the version that installs in
+Flatpak (newer versions seem to have some issues), `uvloop`, and few
+packages required for building other packages.
```

Comment:
The requirement files should be links, e.g. `[requirements.flatpak.txt](requirements.flatpak.txt)`, instead of manually listing again packages here.

Reply:
will do